rest/handler.go

```diff
@@ -484,7 +501,7 @@ func estimateNonceHandler(w http.ResponseWriter, r *http.Request) {
 
 	var resp estimateNonceResp
 	resp.StateID = uint64(idInt)
-	resp.Nonce = pendingNonce
+	resp.Nonce = pendingNonce + 1
```

Comment:
This looks strange to me. This line suggests the first transaction is nonce 1. But in the contract, the first transaction is nonce 0.
But we can unblock this pr, and talk about this more later.

Reply:
Sounds good